api/api-collect/collect/src/main/java/fr/gouv/vitamui/collect/server/service/ArchiveUnitServiceImpl.java

```diff
@@ -119,8 +122,8 @@ private OperationIdDto send(String transactionId, Set<UpdateMultiQuery> updateMu
             log.info("Operation started: {}", operationIdDto);
             return operationIdDto;
         } catch (VitamClientException | InvalidParseOperationException e) {
-            log.error("{}", e);
-            throw new ArchiveUnitUpdateException(e);
+            log.error("Failed to update archive units for transaction: {}", transactionId, e);
+            throw new ArchiveUnitUpdateException("Failed to update archive units: " + e.getMessage(), e);
```

Comment:
Pas nécessaire d'afficher le message, il est contenu dans l'exception.
```suggestion
            throw new ArchiveUnitUpdateException("Failed to update archive units", e);
```